operator/internal/manifests/internal/config/loki-config.yaml

```
@@ -166,9 +166,15 @@ limits_config:
   split_queries_by_interval: 30m
 memberlist:
   abort_if_cluster_join_fails: true
-  bind_port: {{ .GossipRing.Port }}
+  {{- with .GossipRing.BindAddrs }}
+  bind_addr:
+    {{- range . }}
+    - {{ . }}
+    {{- end }}
+  {{- end }}
```

Couldn't this be much simpler because we only have two different possible values (any/default or "pod IP")? (`BindPodIP` is `bool`)

```suggestion
  {{- with .GossipRing.BindPodIP }}
  bind_addr:
    - ${POD_IP}
  {{- end }}
```